src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -12,13 +12,15 @@
 namespace Symfony\Bridge\PhpUnit;
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
+use PHPUnit\Framework\BaseTestListener;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Collects and replays skipped tests.
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
+class SymfonyTestsListener extends BaseTestListener
```

Comment:
How about requiring a proper PHPUnit version? Actually the issue will be that PHPUnit < 5.7 does have another FC layer than 4.8. If we go that route we should enforce 4.8.35, 5.7 or 6